api/src/pcapi/routes/native/v2/account.py

```diff
@@ -15,6 +18,28 @@
 from .serialization import account as serializers
 
 
+@blueprint.native_route("/profile/email_update/status", version="v2", methods=["GET"])
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** Top, faut penser à marquer comme `deprecated` la v1: `@blueprint.api.validate(deprecated=True)
` 
‏